R/get_gpprac_opendata.R

```diff
@@ -0,0 +1,47 @@
+#' GP Practice details from PHS open data
+#'
+#' @return The final data as a [tibble][tibble::tibble-package].
+#' @export
+#'
+get_gpprac_opendata <- function() {
```

Comment:
I've pulled this code from `process_lookup_gpprac` so that i could set it up as a parameter along with spd etc. Hopefully this makes sense to have this set up in this way for targets